src/main/io/osd.c

```diff
@@ -3513,10 +3527,12 @@ static void osdUpdateStats(void)
     if (stats.min_rssi > value)
         stats.min_rssi = value;
 
-    value = osdGetCrsfLQ();
+    value = osdGetCrsfLQ(); {
```

Comment:
Opening `{` here? I think it's a mistake

Reply:
Thanks. Sending out the fix.